src/lando/dockerflow/decorators.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from functools import wraps
+
+# Logging is not yet implemented
+request_logger = logging.getLogger("dockerflow.request_summary.logger")
+
+
+def log_request(view):
+    @wraps(view)
+    def _wrapped_view(self, request, *args, **kwargs):
+        start_time = time.time()
+        response = view(self, request, *args, **kwargs)
+        end_time = time.time()
+
+        summary = {
+            "errno": 0 if response.status_code < 400 else 1,
+            "agent": request.headers.get("User-Agent", ""),
+            "lang": request.headers.get("Accept-Language", ""),
+            "method": request.method,
+            "path": request.path,
+            "code": response.status_code,
+            "t": int(1000 * (end_time - start_time)),
+        }
+
+        request_logger.info("Request Summary: ", extra=summary)
+
+        return response
+
+    return _wrapped_view
+
+
+def disable_caching(view):
+    @wraps(view)
+    def _wrapped_view(self, request, *args, **kwargs):
+        response = view(self, request, *args, **kwargs)
+
+        if request.path.startswith("__"):
```

Comment:
What's the point of having this if statement here? Given that this decorator would have to be explicitly added to views that we want it to affect?

Reply:
Leftover from the original middleware implementation. I'll fix it since it's not needed with the decorators.